.github/workflows/push-vellum-workflows.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
     if: github.event_name == 'pull_request'
     env:
       GITHUB_PRIVATE_KEY: ${{ secrets.VARGASJR_GITHUB_PRIVATE_KEY }}
+      VELLUM_API_KEY: ${{ secrets.VELLUM_API_KEY }}
```

Comment:
Vellum api key works, GitHub private key doesn't. Let's try removing these env keys both here and below since the vercel token above should pull it